src/settings/base.py

```diff
@@ -196,6 +199,8 @@
 # Celery
 # ============================================================================
 CELERY_BROKER_URL = os.environ.get("CLOUDAMQP_URL") or os.environ.get('BROKER_URL')
+# Rabbit/Pyrabbit reference BROKER_URL internally
+BROKER_URL = CELERY_BROKER_URL
```

Comment:
Is there no better workaround than this? If we _have_ to do it this way, we probably need to do this assignment after the `if not CELERY_BROKER_URL:` stuff, right? so this gets set to the correct value?

Reply:
I don't think pyrabbit even needs this? No where in the code does it reference this, unless I am mistaken?

https://github.com/bkjones/pyrabbit/search?q=broker_url&unscoped_q=broker_url

Reply:
why was this marked resolved?

Reply:
I pulled this. I could've sworn PyRabbit was referencing it, but it does not appear to be.